src/main/java/internet/shop/service/ShoppingCartService.java

```diff
@@ -0,0 +1,18 @@
+package internet.shop.service;
+
+import internet.shop.model.Product;
+import internet.shop.model.ShoppingCart;
+
+public interface ShoppingCartService {
+    ShoppingCart create(ShoppingCart shoppingCart);
+
+    ShoppingCart addProduct(ShoppingCart shoppingCart, Product product);
+
+    ShoppingCart deleteProduct(ShoppingCart shoppingCart, Product product);
```

Comment:
```suggestion
    boolean deleteProduct(ShoppingCart shoppingCart, Product product);
```
be more attentive with task description